src/main/kotlin/no/nav/familie/tilbake/api/PerioderController.kt

```diff
@@ -33,7 +33,7 @@ class PerioderController(
     @Rolletilgangssjekk(
         Behandlerrolle.SAKSBEHANDLER,
         "Sjekker om perioder er like - unntatt dato og beløp",
-        AuditLoggerEvent.UPDATE,
+        AuditLoggerEvent.ACCESS,
```

Comment:
Enig i at dette er en Access, men er det egentlig Behandlerrolle.SAKSBEHANDLER vi trenger fikse for at veiledere skal kunne se på behandlig?